invenio_cli/commands/local.py

```diff
@@ -183,9 +186,47 @@ def run_worker(self, celery_log_file=None, celery_log_level="INFO"):
                 celery_log_file,
             ]
 
+        processes = []
         proc = popen(celery_command)
         self._handle_sigint("Celery worker", proc)
         click.secho("Worker running!", fg="green")
+        processes.append(proc)
+
+        if jobs_scheduler:
+            processes.extend(self.run_jobs_scheduler(celery_log_file, celery_log_level))
+
+        return processes
+
+    def run_jobs_scheduler(self, celery_log_file=None, celery_log_level="INFO"):
+        """Run Celery beat scheduler for jobs."""
+        # Jobs scheduler is only available in RDM v13+
+        version = rdm_version()
+        if not version or version[0] < 13:
```

Comment:
Nice!

Reply:
I thought of a couple of alternatives:

- have this information coming [from `CLIConfig`](https://github.com/inveniosoftware/invenio-cli/blob/77e871406449717c4f86d7c47ca1eacc58953af1/invenio_cli/helpers/cli_config.py#L30), so that one e.g. could explicitly set something like `version = 13` in their `.invenio` and use the whole "Pipfile/pyproject.toml detection" dance as a fallback...
- get the version from the package manager (i.e. run `uv pip show invenio-app-rdm` or the `pipenv` equivalent), but I think that was the case in the past but is flakey since you might not have at all points the dependencies installed...
- get from the lockfiles. has similar caveats as the above point, where the lockfile might not have been generated yet

Reply:
> > i would prefer to crash directly, if version is not a valid value
> 
> Maybe you prefer to crash with a specific message like "Cannot detect app rdm version" or similar?

yes, that would be the best option. since the app-rdm version is a important information for invenio-cli, it should fail early in the usage of `invenio-cli`

Reply:
> I thought of a couple of alternatives:
> 
>     * have this information coming [from `CLIConfig`](https://github.com/inveniosoftware/invenio-cli/blob/77e871406449717c4f86d7c47ca1eacc58953af1/invenio_cli/helpers/cli_config.py#L30), so that one e.g. could explicitly set something like `version = 13` in their `.invenio` and use the whole "Pipfile/pyproject.toml detection" dance as a fallback...

that was my attempt to solve my problem. i added a value to the `.invenio.private` file and added a method `CLIConfig`

Reply:
That would require that on each upgrade, we ask users to change the `.invenio` file and add/bump the version, right?

Reply:
no, that would mean, that users which don't add `invenio-app-rdm` into their `pyproject.toml` file of their instance, they have to change it in their .invenio.private file

which will not be that often, and if you customize your setup that much means you are, in my point of view, a power user, and those should have more options.

so my suggestions is not to replace the current way of doing it, only to add a new place where the app-rdm version could be picked up